src/pages/big-map/MainLayout.tsx

```diff
@@ -98,3 +105,56 @@ export default function MainLayout() {
     </div>
   );
 }
+
+const aboutInformationCss = {
+  root: css`
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    gap: 5px;
+    align-items: center;
+  `,
+  link: css`
+    color: rgb(150, 150, 150);
+    &:hover {
+      color: rgb(0, 188, 212);
+    }
+  `,
+  separator: css`
+    border-bottom: 1px solid gray;
+    width: 15px;
+    height: 1px;
+    margin: 10px 0px;
+  `,
+};
+
+function AboutInformation() {
```

Comment:
Please move this component to a new file

Reply:
I mean a new file in the big-map folder.
The about dialog will be specific to this application.